docs/ref/files/storage.txt

```diff
@@ -11,7 +11,30 @@ Django provides convenient ways to access the default storage class:
 
 .. data:: storages
 
-    Storage instances as defined by :setting:`STORAGES`.
+    Mapping of aliases to storage instances as defined by :setting:`STORAGES`.
+
+    ``storages`` has a dictionary attribute ``backends``, which at startup is
+    the raw value provided to :setting:`STORAGES`.
+
+    Third-party packages providing storages may allow users to optionally
+    customize a storage for a known alias, using ``backends`` to provide a
+    default::
```

Comment:
```suggestion
    default. Here's an example implementation of such an utility::
```

Maybe that's superfluous. As a reader it wasn't immediately clear to me that `get_storage` is a thing which should be added to a third party package. The additional clarification may be helpful.

Also, I learned another thing today :-)

Reply:
I agree, I am feel like this example should be over here: https://docs.djangoproject.com/en/5.1/howto/custom-file-storage/#use-your-custom-storage-engine